src/sql/workbench/browser/designer/designer.ts

```diff
@@ -346,12 +370,41 @@ export class Designer extends Disposable implements IThemable {
 			}
 			if (edit.type === DesignerEditType.Add) {
 				// For tables in the main view, move focus to the first cell of the newly added row, and the properties pane will be showing the new object.
```

Comment:
the comments need to be updated to move focus to first editable cell of the newly added row.